tests/unittests/testperformance.cpp

```diff
@@ -30,6 +30,7 @@
 #include <string.h>
 #include <ctime>
 
+#include "CGlobalExporterConfigs.h"
```

Comment:
Why was this added?

Reply:
This is something I had to add in order to be able to access the fuser instance in line 247 and 248 - the problem is that I figured out that I needed to include this be trial and error - without this include, that is also used in tigl.cpp, accessing the fuser doens't work. This needs to be investigated.

Reply:
Have you tried `"CTiglFusePlane.h"`? 

`CGlobalExporterConfigs.h` includes `CTiglCADExporter.h` which includes `CTiglFusePlane.h`, but let's just include what we need.

Reply:
I just tried out your suggestion and it works, great! Thanks for the insight!